src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    public void drawFigure();
+}
```

Comment:
Interface should be named `Drawable`
Method shouldn't be tied up to figures and have abstract name like `draw`

https://howtodoinjava.com/java/basics/java-naming-conventions/